IPython/html/base/handlers.py

```diff
@@ -138,6 +138,10 @@ def base_url(self):
         return self.settings.get('base_url', '/')
 
     @property
+    def default_url(self):
+        return self.settings.get('default_url', '')
```

Comment:
I think this has the same infinite redirect issue as `/`, but I'm satisfied that it can't happen in IPython.